packages/react-components/react-combobox/src/ComboboxBase/ComboboxBase.types.ts

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import type { PositioningShorthand } from '@fluentui/react-positioning';
+import type { OptionValue, OptionCollectionState } from '../utils/OptionCollection.types';
+import { SelectionEvents, SelectionProps, SelectionState } from '../utils/Selection.types';
+
+/**
+ * ComboboxBase Props
+ */
+export type ComboboxBaseProps = SelectionProps & {
```

Comment:
Hm, there is Listbox already in the package, and this includes the combobox functionality and semantics more than just listbox (and actually doesn't have much listbox stuff, since that's already in Listbox).

Reply:
though to be fair, most people probably don't know the difference between combobox and listbox anyway, so maybe that doesn't matter 😂

Reply:
🤣 Clearly I don't 🤣

My thought is that if this functionality will be shared with Dropdown it feels odd to call it "ComboboxBase". `SelectableListBase`? 🤷

Follow up question: will this base component be part of the public API? If not, I don't want to get hung up on the name since we can change it later of it's internal only.

Reply:
oooo, I like `SelectableListBase`! I'd want to make sure it doesn't seem like it's only the list, since I assume we'll have other SelectableListBases for things like List/Grid, but since this is internal to react-combobox, I don't think it matters too much. (and yup, not exported at all, internal-only 😄)